Makefile

```
@@ -222,8 +222,7 @@ kplutus_files := uplc.md \
                  uplc-polymorphic-builtins.md \
                  uplc-string.md \
                  uplc-data-builtins.md \
-                 uplc-hash.md \
-		 uplc-discharge.md
+				 uplc-discharge.md
```

```suggestion
                 uplc-discharge.md
```
(nitpick) make sure you're being consistent with the whitespace being used. You generally won't see tabs anywhere in RV's codebase.

---

Oh, I see.